proto/tendermint/types/canonical.proto

```diff
@@ -34,4 +34,9 @@ message CanonicalVote {
   CanonicalBlockID          block_id  = 4 [(gogoproto.customname) = "BlockID"];
   google.protobuf.Timestamp timestamp = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   string                    chain_id  = 6 [(gogoproto.customname) = "ChainID"];
+  CanonicalVoteExtension    vote_extension = 7;
+}
+
+message CanonicalVoteExtension {
```

Comment:
@ebuchman Should we make this field wrapped in "CanonicalVoteExtension" to mirror "CanonicalBlockID" as is implemented right now, or should `CanonicalVote` directly have a field called `app_data_signed`.

Reply:
I found that `CanonicalVote` is also the field that has to be included in the `CommitSig`. It has been renamed to `VoteExtensionSigned` in a recent commit.